datadog.conf.example

```diff
@@ -48,24 +47,23 @@ api_key:
 # collector_profile_interval: 20
 
 # use unique hostname for GCE hosts, see http://dtdg.co/1eAynZk
-gce_updated_hostname: yes
+# gce_updated_hostname: yes
```

Comment:
Unfortunately you can't do that. As we want to keep the old defaults for existing installations (for compatibility purposes) but we want new installations to have this flag enabled